src/graph.rs

```diff
@@ -883,6 +883,12 @@ impl<N, E, Ty, Ix> Graph<N, E, Ty, Ix>
         &self.edges
     }
 
+    /// Return the internal arrays, consuming the graph.
+    pub fn destructure(self) -> (Vec<Node<N, Ix>>, Vec<Edge<E, Ix>>)
```

Comment:
I was wondering about that, I'm assuming you needed it in the consuming form. We can't really offer both in the same function unfortunately..